Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Set the base image
+FROM openjdk:8-jre-alpine
+
+# Dockerfile author / maintainer
+MAINTAINER SourceLag.org <stephen.powis@gmail.com>
+
+# This copies binary distribution inside the image
+RUN mkdir -p /app
+ADD kafka-webview-ui/target/kafka-webview-ui-1.0.0-bin.zip /app
```

Comment:
maybe a checkout from github to build / or a web-call to a release from the release page?